### DISCUSSION TITLE
Remote for using MA in shower

### DISCUSSION THREAD

**Discussion by MoreIs1** [music-assistant/support#2152] [Q&A]
I want to misuse a Tradfri 5-button remote to use as a remote in the shower. What commands can I use for play/pause (power), volume up/down (dim down/up) and previous/next track (color up/down)? I'm using the Blueprint [ZHA - IKEA TRADFRI - 5 Button Remote - Custom Actions](https://community.home-assistant.io/t/zha-ikea-tradfri-5-button-remote-custom-actions/284720).

**Comment by OzGav** [comment]
You can use the normal HA media services https://www.home-assistant.io/integrations/media_player/

**Reply by MoreIs1** [reply]
I am a starter at Home Assistant. I have created some code within the Blueprint, but you have to set a target. And to my common sense the target must be Music Assistant, not (one of) the speaker(s). What is the entity for Music Assistant? I can't find it...

alias: Badkamer Tradfri media remote
description: ""
use_blueprint:
  path: niro1987/zha_ikea_tradfri_5button_remote_custom.yaml
  input:
    remote: b76dac3aab7a061a2cf5687e34a0677e
    power_short:
      - service: media_player.media_play_pause
        data: {}
    dimup_short:
      - service: media_player.volume_up
        metadata: {}
        data: {}
    dimdown_short:
      - service: media_player.volume_down
        metadata: {}
        data: {}
    colorup_short:
      - service: media_player.media_next_track
        metadata: {}
        data: {}
    colordown_short:
      - service: media_player.media_previous_track
        metadata: {}
        data: {}

**Reply by OzGav** [reply]
Have you added the HA integration from HACS? If you have then you will see the MA entities.

**Reply by MoreIs1** [reply]
Yes, added it from HACS and I had been fiddling around. Also renamed the bathroom speaker. Turns out you do have to target the speaker and not Music Assistant. And it took some time before the correct entities appeared again. Problem is fixed and I have a nice remote for a very nice media player! ;)
